megamek/src/megamek/client/ui/swing/lobby/MekForceTreeRenderer.java

```diff
@@ -124,8 +125,9 @@ public String getToolTipText(MouseEvent event) {
         return null;
     }
 
-    private void setIcon(Image image, int size) {
-        setIcon(new ImageIcon(image.getScaledInstance(-1, size, Image.SCALE_SMOOTH)));
+    private void setIcon(Image image, int height) {
+        int width = height * image.getWidth(null) / image.getHeight(null);
+        setIcon(new ImageIcon(ImageUtil.getScaledImage(image, width, height)));

```

Comment:
We've seen errors in the past where `getWidth`/`getHeight` returns `0` or `-1`. Lets check for `image.getHeight(null) > 0` before proceeding.